devtools/test_dashboard/devtools.js

```diff
@@ -19,7 +19,7 @@ var Tabs = {
         Plotly.setPlotConfig({
 
             // use local topojson files
-            topojsonURL: '../../vendor/topojson/',
+            topojsonURL: '/node_modules/sane-topojson/dist/',
```

Comment:
Or it should be 
```js
topojsonURL: '../../node_modules/sane-topojson/dist/',
```
?

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5492/commits/815d0471c8e3af473a21bfdfdd48b3162091b4b4.